server/service/src/sync/api/core.rs

```diff
@@ -40,15 +39,28 @@ fn generate_headers(hardware_id: &str) -> HeaderMap {
     headers
 }
 
+#[derive(Error, Debug)]
+pub enum SyncApiV5CreatingError {
```

Comment:
Added this so that we can return invalid url in API layer, i've noticed quite a few times when url was wrong that we would just get generic error from API and then in turn generic error in front end, now it would say invalid url

Reply:
In head PR you can see this in: link here